packages/plugins/script/write-constants.ts

```diff
@@ -93,7 +93,7 @@ const writeConstants = async () => {
     },
   }
 
-  const contractInfo = CHUGSPLASH_CONTRACT_INFO.map(
+  const contractInfo = getChugSplashConstants(31337).map(
```

Comment:
We don't need the funding-related contracts in here, so we use a default chain id and then filter out the funding-related contracts below so they don't even make it into the constants file.